### DISCUSSION TITLE
FederatedCredentials

### DISCUSSION THREAD

**Discussion by rajyraman** [microsoftgraph/msgraph-bicep-types#84] [Q&A] [answered]
I thought I will try FederatedCredentials, but I am getting build error. Is this ready, or is there any way to try nightly builds?

![image](https://github.com/microsoftgraph/msgraph-bicep-types/assets/5035266/9b318af3-8401-4b71-b343-7b6caff017b2)

**Comment by jason-dou** [comment, marked as answer]
Thanks @rajyraman - FederatedIdentityCredentials is currently not ready in Bicep yet, and we hope to have it available with the Bicep official release in April. Will provide an update here once it's available in a nightly build.
